src/otx/algorithms/common/adapters/mmcv/hooks/xpu_optimizer_hook.py

```diff
@@ -0,0 +1,38 @@
+"""Custom Optimizer Hook for mixed precision training on XPU."""
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+from typing import Optional, Union
+
+from mmcv.runner.hooks import HOOKS, Fp16OptimizerHook
+
+from otx.algorithms.common.adapters.torch.amp import XPUGradScaler
+
+
+@HOOKS.register_module()
+class XPUOptimizerHook(Fp16OptimizerHook):
```

Comment:
If it's for bfp16 mixed precision training, how about chainging class name to `Bfp16XPUOptimizerHook`?

Reply:
Done.